Sources/ViewInspector/ViewSearch.swift

```diff
@@ -378,13 +378,11 @@ private extension UnwrappedView {
             if name.hasPrefix(ViewType.Popover.standardModifierName) {
                 return "popover"
             }
-            if let inspectable = view as? InspectableProtocol {
-                let missingObjects = inspectable.missingEnvironmentObjects
-                if missingObjects.count > 0 {
-                    return InspectionError
-                        .missingEnvironmentObjects(view: name, objects: missingObjects)
-                        .localizedDescription
-                }
+            let missingObjects = EnvironmentInjection.missingEnvironmentObjects(for: view)
```

Comment:
I'm not sure what the implications are of checking for missing environment objects are in all views instead of only views that conform to `Inspectable`.